tests/remote/register/test_register.py

```diff
@@ -64,7 +64,7 @@ def test_get_base_docker_image_gpu():
     && apt-get install -y software-properties-common\
     && add-apt-repository -y ppa:deadsnakes/ppa\
     && apt-get -y upgrade\
-    && apt-get install -y python3.11 python3.11-venv python3-pip\
+    && apt-get install -y python3.11 python3.11-venv python3-pip \
```

Comment:
```suggestion
    && apt-get install -y python3.11 python3.11-venv python3-pip\
```

Reply:
This whitespace is required to make the test pass, actually :)

Reply:
🤔  weird

Reply:
That's the place where the binary dependencies would be if there were defined

Reply:
ah ok make sense,